build.sh

```diff
@@ -9,9 +9,9 @@ make
 
 cd python-package
 if command -v python2; then
-    sudo python2 setup.py install
```

Comment:
Remove "sudo" may cause "Permission denied" on some systems.

Reply:
Hi @aksnzhy, Thanks for the comment. For the sudo priviledge, could we add sudo to the bash script instead? This assumption might be too strong if the user does not have sudo priviledge (e.g. on VM) or they are using conda environment that do not require sudo.

Reply:
You are right. We can move this to bash.